R/convertExpertGuess.R

```diff
@@ -3,21 +3,28 @@
 #' @param x unconverted magpie object from read-script
 #' @param subtype Type of data that are converted.
 #'
-#'
 convertExpertGuess <- function(x, subtype) {
-  if (subtype == "costsTradePeFinancial") {
-    # use data for each country that belongs to a region
-    # No weighting for spatial aggregation
-    out <- toolAggregate(x, toolGetMapping(
-      type = "regional", name = "regionmappingH12.csv",
-      returnPathOnly = TRUE, where = "mappingfolder"
-    ),
-    weight = NULL
-    )
-  } else if (subtype %in% c("taxConvergenceRollback", "subConvergenceRollback")) {
+
+  if (subtype %in% c(
+    "capacityFactorRules",
+    "costsTradePeFinancial",
+    "subConvergenceRollback",
+    "taxConvergenceRollback"
+  )) {
+
+    # source data for these types is in H12 regions
+    # use the region data for each country that belongs to the region
+    # no weighting for spatial aggregation
```

Comment:
I thoght that we would like to add "subConvergenceRollback" and "taxConvergenceRollback" in the other tables? Or am I wrong?